chapters/ch46.md

```diff
@@ -1,51 +1,52 @@
 
---> _replace THIS LINE by your translation for the above line_
+Để áp dụng học tăng cường, bạn phải phát triển một "hàm điểm thưởng" *R*(.) nó sẽ cho một điểm số để đo xem mỗi quỹ đạo *T* tốt như thế nào. Lấy ví dụ, nếu *T* kết thúc bằng việc trực thăng bị rơi, thì điểm thưởng có thể sẽ là *R(T)* = -1.000-một điểm thưởng âm rất lớn. Một quỹ đạo *T* kết thúc bằng việc trực thăng hạ cánh an toàn có thể sẽ cho *R(T)* dương với giá trị chính xác phụ thuộc vào việc hạ cánh êm ái như thế nào. Hàm điểm thưởng *R*(.) thường được chọn thủ công để định lượng mức độ mong muốn của những quỹ đạo *T* khác nhau. Nó phải đánh đổi giữa những đặc tính như mức độ xóc của cú hạ cánh, trực thăng có hạ cánh đúng vị trí mong muốn không, quá trình hạ độ cao có nhiều biến động đối với hành khách không, và vân vân. Thiết kế một hàm điểm thưởng tốt không hề dễ dàng.
 
 > Given a reward function ​*R(T)*, ​the job of the reinforcement learning algorithm is to control the helicopter so that it achieves max​<sub>*T*</sub>​*R(T)*. ​However, reinforcement learning algorithms make many approximations and may not succeed in achieving this maximization.
 
---> _replace THIS LINE by your translation for the above line_
+Cho một hàm điểm thưởng *R(T)*, công việc của thuật toán học tăng cường là để điều khiển trực thăng sao cho nó đạt được điểm thưởng cao nhất max<sub>*T*</sub>*R(T)*. Tuy nhiên, thuật toán học tăng cường có nhiều phép ước lượng và có thể sẽ không thành công trong việc tối ưu này.
 
 > Suppose you have picked some reward ​*R(.)​* and have run your learning algorithm. However,
 > its performance appears far worse than your human pilot—the landings are bumpier and
 > seem less safe than what a human pilot achieves. How can you tell if the fault is with the
 > reinforcement learning algorithm—which is trying to carry out a trajectory that achieves
 > max​<sub>*T*</sub>​*R(T)*—​ or if the fault is with the reward function—which is trying to measure as well as specify the ideal tradeoff between ride bumpiness and accuracy of landing spot?
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử bạn đã có vài điểm thưởng *R(.)* và đã chạy thuật toán học của bạn. Tuy nhiên chất lượng của nó còn thua xa chất lượng của con người-hạ cánh xóc hơn và trông kém an toàn hơn so với chất lượng của phi công. Làm sao để bạn biết được liệu đó có phải là lỗi của thuật toán học tăng cường-cái mà đang đang cố thực thi một quỹ đạo để tối đa max<sub>*T*</sub>*R(T)*- hay là lỗi của hàm điểm thưởng-cái mà đang cố đo cũng như xác định mức đánh đổi tối ưu giữa độ xóc của chuyến bay và độ chính xác của vị trí hạ cánh. 
```

Comment:
Đoạn này bản gốc có vđề về cách viết nên mình cần chỉnh chút cho dễ đọc ở bản dịch:
> Giả sử bạn đã có vài điểm thưởng *R(.)* và đã chạy thuật toán học của bạn. Tuy nhiên chất lượng của nó còn thua xa chất lượng của con người (ví dụ: hạ cánh xóc hơn và có vẻ kém an toàn hơn so với chất lượng của phi công). Làm sao để bạn biết được liệu đó có phải là lỗi của thuật toán học tăng cường - cái mà đang đang cố thực thi một quỹ đạo để tối đa max<sub>*T*</sub>*R(T)* - hay là lỗi của hàm điểm thưởng (cái mà đang cố đo cũng như xác định mức đánh đổi tối ưu giữa độ xóc của chuyến bay và độ chính xác của vị trí hạ cánh).

Reply:
-  kém an toàn hơn _so với chất lượng của phi công_ --> _so với người lái._

Reply:
- thừa chữ "đang" trong "cái mà đang ~~đang~~ cố thực thi"

Reply:
> Đoạn này bản gốc có vđề về cách viết nên mình cần chỉnh chút cho dễ đọc ở bản dịch:

Thanks bạn. Trong đoạn này có 2 ý ".. - cái mà ..", một cái bạn để trong ngoặc một cái để ngoài nên mình có sửa lại cho thống nhất hơn.